cdap-docs/reference-manual/source/http-restful-api/service.rst

```diff
@@ -69,6 +69,80 @@ The response body will contain a JSON-formatted list of the existing system serv
 See :ref:`downloading System Logs <http-restful-api-logging_downloading_system_logs>` for
 information and an example of using these system services.
 
+Restarting System Service Instances
+------------------------------------
+
+To restart all instances of a system service in CDAP, you can issue an HTTP POST request to the URL::
+
+  POST <base-url>/system/services/<service-id>/restart
+
+You can also restart a particular instance of a system service in CDAP, using the instance id, by issuing
+an HTTP POST request to the URL::
+
+  POST <base-url>/system/services/<service-id>/instances/<instance-id>/restart
+
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+   * - ``<service-id>``
+     - Name of the service which instances to be restarted
+   * - ``<instance-id>``
+     - Specific instance id need to be restarted
```

Comment:
How do you get instance-ids? Should we reference that?
Also: I'm confused by " instance ids are not unique among all services." Is an instance ID unique among all instances, or is the combination of service plus instance ID that is unique?